invenio_vocabularies/resources/resource.py

```diff
@@ -68,27 +69,6 @@ class VocabulariesResourceConfig(RecordResourceConfig):
     }
 
 
-#
-# Decorators
```

Comment:
None of these implements anything new with respect to the original ones in flask-resources.